frontend/packages/console-shared/src/hooks/useOperator.ts

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { consoleFetchJSON as coFetchJSON } from '@console/dynamic-plugin-sdk/src/utils/fetch';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const useOperator = (
+  operatorName: string,
+  operatorNamespace: string,
+): [K8sResourceCommon[], boolean, string] => {
+  const [operands, setOperands] = React.useState<K8sResourceCommon[]>([]);
```

Comment:
I think operands is a misnomer if I'm follwing the code correctly. This would contain a single PackageManifest, right?